app/src/features/Batches/BatchListItem.tsx

```diff
@@ -0,0 +1,144 @@
+
+type BatchListItemType = {
+  batch: Batch;
+};
+
+const useStyles = makeStyles((theme) => ({
+  listItem: {
+    marginBottom: theme.spacing(0.5),
+    display: "flex",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: theme.spacing(1.3, 2, 1, 2),
+    flexWrap: "wrap",
+  },
+  listItemActions: {
+    display: "flex",
+    alignItems: "center",
+    flexWrap: "wrap",
+  },
+  margin: {
+    marginRight: theme.spacing(1),
+  },
+  title: {
+    display: "flex",
+  },
+}));
+
+const createKibanaLink = (batchCreation: string, batchEnd: string) => {
+  return `${KIBANA_URL}/app/kibana#/discover?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'${batchCreation}',to:'${batchEnd}'))&_a=(columns:!(_source),filters:!(),interval:auto,query:(language:kuery,query:error),sort:!())`;
+};
+
+const BatchListItem = ({ batch }: BatchListItemType): JSX.Element => {
+  const classes = useStyles();
+  const { t } = useTranslation();
+  const batchEnd = batch.completed_at ?? batch.canceled_at;
+
+  const isBatchInProgress = !batch.completed_at && !batch.canceled_at;
+  const hasBatchCompletedWithErrors =
+    batch.completed_at && batch.errors.length > 0;
+  const hasBatchCompletedWithoutError =
+    batch.completed_at && batch.errors.length === 0;
+  const isBatchRunning = !batch.canceled_at && !batch.completed_at;
+  const hasBatchStopped = batch.completed_at || batch.canceled_at;
+
+  const getBatchDuration = (batch: Batch) => {
+    if (batchEnd) {
+      const start = moment(batch.created_at);
+      const end = moment(batchEnd);
+      const diff = moment.duration(end.diff(start));
+      return `${diff.years() ? `${t("year", { count: diff.years() })} ` : ""}${
+        diff.months() ? `${t("month", { count: diff.months() })} ` : ""
+      }${diff.days() ? `${t("day", { count: diff.days() })} ` : ""}${
+        diff.hours() ? `${t("hour", { count: diff.hours() })} ` : ""
+      }${diff.minutes() ? `${t("minute", { count: diff.minutes() })} ` : ""}${
+        diff.seconds() ? `${t("second", { count: diff.seconds() })} ` : ""
+      }`;
+    }
+  };
+
+  const getCreatedAtDate = (date: string) =>
+    new Date(date).toLocaleString().split(",").join(" -");
+
+  const handleBatchRetry = (
+    e: React.MouseEvent<HTMLButtonElement, MouseEvent>
+  ) => {
+    e.stopPropagation();
+    // TODO: batch retry is not implemented yet
+    console.log("Retrying a batch is not yet implemented !");
+  };
+
+  return (
+    <Paper
+      key={batch.id}
+      className={classes.listItem}
+      variant="outlined"
+      elevation={2}
+    >
+      <div>
+        <div className={classes.title}>
+          <Typography variant="subtitle2">
+            {isBatchInProgress && t("batchInProgress")}
+            {hasBatchCompletedWithErrors && (
+              <>
+                {t("batchErrors", {
+                  count: batch.errors.length,
+                })}{" "}
+                {batchEnd && (
```

Comment:
we still want to see errors on kibana if `batchEnd` doesn't exist (ie: batch is still running)